src/protocols/secure_channel/RendezvousParameters.h

```diff
@@ -93,6 +97,28 @@ class RendezvousParameters
     bool HasConnectionObject() const { return false; }
 #endif // CONFIG_NETWORK_LAYER_BLE
 
+    bool HasMRPConfig() { return mMRPConfig.HasValue(); }
```

Comment:
nit: const method too? It sems we do that for other Getter/Has (above and below)